oci_runtime.rst

```diff
@@ -22,64 +22,97 @@ with containers on Linux, follow the OCI standards.
-2. The :ref:`OCI Command Group section <oci_command>` of this page documents the
+addressed, {Singularity}'s default native runtime is not fully OCI compatible.
+However, over time, {Singularity} has continuously improved compatibility with
+OCI standards so that the majority of OCI container images can be run using it.
+
+In {Singularity} 4.0, a new OCI-mode is now fully supported. This mode, enabled
+via the ``--oci`` CLI option or ``oci mode`` directive in ``singularity.conf``, uses
+an OCI low-level runtime to execute containers that achieve compatibility that is
+not possible with {Singularity}'s native runtime. The familiar ``singularity``
+command line interface is maintained and unique features such as the SIF image format
+continue to offer benefits for HPC environments.
+
+OCI compatibility is discussed in three areas of this guide:
+
+#. The :ref:`OCI-mode section <oci_mode>` of this page introduces the new
+   OCI-mode (``--oci``), which runs OCI / Docker containers using a true OCI
+   low-level runtime.
+#. The :ref:`Support for Docker <singularity-and-docker>` page discusses
+   limitations, compatibility options, and best practices for running OCI /
+   Docker containers with {Singularity}'s default runtime.
+#. The :ref:`OCI Command Group section <oci_command>` of this page documents the
    ``singularity oci`` commands, which provide a low-level means to run
    {Singularity} SIF containers with a command line that matches other OCI
    runtimes.
-3. The :ref:`Support for Docker <singularity-and-docker>` page discusses
-   limitations, compatibility options, and best practices for running OCI /
-   Docker containers with {Singularity}'s default runtime.
 
 ********************
 OCI Mode (``--oci``)
 ********************
 
-Users can run an OCI / Docker container in its native format by adding the
-``--oci`` flag to a ``run / shell /exec`` command:
+OCI-mode, enabled with the ``--oci`` command line option, or the ``oci mode``
+directive in ``singularity.conf``, is now fully supported from {Singularity}
+4.0. When OCI-mode is enabled:
 
-.. code::
+- OCI containers are executed using ``crun`` or ``runc`` as the low-level
+  runtime, for true OCI runtime compatibility.
+- Default behaviour (of bind mounts etc.) is comparable to using the
+  ``--compat`` mode with the native runtime.
+- Containers retrieved from OCI sources are encapsulated within a single OCI-SIF
+  file, maintaining the benefits of SIF while avoiding a full conversion into
+  {Singularity}'s native container format.
 
-  $ singularity shell --oci docker://ubuntu
-  2023/02/06 11:00:10  info unpack layer: sha256:677076032cca0a2362d25cf3660072e738d1b96fe860409a33ce901d695d7ee8
-  Singularity> echo "Hello OCI World!"
-  Hello OCI World!
+Users are encouraged to employ OCI-mode when their primary use-case for
+{Singularity} is to run existing containers from Docker Hub or other OCI
+registries. Behavior will more closely match that described for docker than with
```

Comment:
```suggestion
registries. Behavior will more closely match that described for Docker than with
```

(In most of the rest of the document, this is capitalized, unless it's being used as the transport field in a URI of course.)